lib/dal/src/management/prototype.rs

```diff
@@ -409,6 +420,27 @@ impl ManagementPrototype {
         self.description.as_deref()
     }
 
+    pub async fn kind_by_id(
+        ctx: &DalContext,
+        id: ManagementPrototypeId,
+    ) -> ManagementPrototypeResult<ManagementFuncKind> {
+        let func_id = Self::func_id(ctx, id).await?;
+        let func = Func::get_by_id(ctx, func_id).await?;
+        // TODO: Make Management Func Kinds a real thing
```

Comment:
I think this will eventually bite us, especially when multi-cloud comes into play. Good call out